coalib/bearlib/aspects/Redundancy.py

```diff
@@ -0,0 +1,146 @@
+from coalib.bearlib.aspects import Aspect, AspectDocumentation
+
+
+class Redundancy(Aspect):
+    docs = AspectDocumentation(
+        definition="This aspect describes redundancy in your source code.",
+        cause="Redundancies are often caused by forgetting to remove "
+              "something during refactorings.",
+        example="if False:\n    do_something()",
+        example_language="python",
+        importance_reason="Redundant code makes your code harder to read and "
+                          "understand.",
+        fix_suggestions="Redundant code can usually be removed without "
+                        "consequences."
+    )
+
+    class Clone(Aspect):
+        docs = AspectDocumentation(
+            definition="""
+            Code clones are different pieces of source code in your
+            codebase that are very similar.
+            """,
+            cause="""
+            Clones usually result out of copy and pasting.
+            """,
+            example="""
+            if input=="main":
+                print("This is main.")
+            elif input=="other":
+                print("This is other.")
+            """,
+            example_language='python',
+            importance_reason="""
+            Code clones are a problem because they bloat up your code base and
+            bugs usually occur in all the duplicated fragments.
+            """,
+            fix_suggestions="""
+            Usually code clones can be simplified to only one occurrence. In a
+            lot of cases, both just use different values or variables and can
+            be reduced to one function called with different parameters or
+            loops.
+            """
+        )
+
+        MIN_CLONE_TOKEN = (
+            "The number of tokens that have to be equal for it to be detected "
+            "as a code clone.",
```

Comment:
maybe we can be a bit more verbose here, @Asnelchristian didn't you want to write a documentation class kind of thing for this as well?